### ISSUE TITLE
Request: add polylith tool to the list of tools that can be installed

### ISSUE THREAD

**Issue by licht1stein** [DeLaGuardo/setup-clojure#95]
Hi, thanks for the action!

Would it be too difficult to add the `poly` tool to the list of tools that can be automatically installed?

https://github.com/polyfy/polylith

**Comment by DeLaGuardo** [comment]
Good idea, thanks! I'll check what would be the requirements. If I recall correctly, polylith relies on some standalone cli tool.

**Comment by licht1stein** [comment]
Yes, here are the docs:

https://polylith.gitbook.io/poly

**Comment by licht1stein** [comment]
And install instructions: 

https://polylith.gitbook.io/poly/install/install/install-on-linux

**Comment by russellwhitaker** [comment]
@DeLaGuardo would you like someone like @licht1stein or myself to provide the patch to add Polylith? I could be motivated to do this myself, since I need it for work.

**Comment by tikto29** [comment]
russellwhitaker commented on Nov 1, 2024
russellwhitaker
on Nov 1, 2024
@DeLaGuardo would you like someone like @licht1stein or myself to provide the patch to add Polylith? I could be motivated to do this myself, since I need it for work.
